Source/Urho3D/UI/ProgressBar.cpp

```diff
@@ -42,7 +42,6 @@ ProgressBar::ProgressBar(Context * context) :
         value_(0.0f),
         showPercentText_(true)
 {
-    SetEnabled(false);

```

Comment:
Bit unsure about this one. It might have been meant to be true (since it changes the default below), in which case there should be a SetPassthrough(false) here.